lib/internal/main/mksnapshot.js

```diff
@@ -134,6 +151,16 @@ function main() {
   } else {
     serializeMainFunction(requireForUserSnapshot, filename, dirname);
   }
+
+  addSerializeCallback(() => {
+    if (ObjectPrototypeHasOwnProperty(Error, 'stackTraceLimit')) {
+      stackTraceLimitDesc =
+          ObjectGetOwnPropertyDescriptor(Error, 'stackTraceLimit');
```

Comment:
```suggestion
    stackTraceLimitDesc = ObjectGetOwnPropertyDescriptor(Error, 'stackTraceLimit');

    if (stackTraceLimitDesc !== undefined) {
      ObjectSetPrototypeOf(stackTraceLimitDesc, null);
```

Reply:
Why are we mutating the prototype?

Reply:
Oh never mind, I checked out the primordials guide 👍